test/hotspot/jtreg/gc/cslocker/TestCSLocker.java

```diff
@@ -51,7 +51,6 @@ public static void main(String args[]) throws Exception {
 
         // check timeout to success deadlocking
         while(System.currentTimeMillis() < startTime + timeout) {
-            System.out.println("sleeping...");
```

Comment:
I think some comments (one cannot run any gc-triggering code, e.g. println) here would be nice. It's super tempting to add some innocent debug-prints before suspending the current thread, while extending/fixing this test case in the future.